website/docs/reference/feature-toggle-variants.md

```diff
@@ -1,13 +1,18 @@
 ---
-title: Feature Toggle Variants
+title: Feature Toggle Variants (deprecated)
 ---
 :::info Availability
 
 **Feature toggle variants** were first introduced in Unleash 3.2.
+
 In Unleash 4.21, variants were updated to be **environment-dependent**, meaning the same feature could have different variant configurations in different environments.
 
+Unleash 6.0 deprecates environment-dependent variants in favor of the [strategy variants](./strategy-variants.md). New Unleash instances and those who never used environment variants will
```

Comment:
I would pull this higher and put it into a warning-like box or something. So that is first thing that people see. Basically we want all new users to know that this is not worth reading, you should not use it.